app/Models/Post.php

```diff
@@ -264,8 +264,12 @@ public function getQuantity()
     /**
      * Apply the given tag to this post.
      */
-    public function tag($tagName)
+    public function tag($tag)
```

Comment:
I think this might break other uses of this where we don't pass in a slug.

I think we should actually check if we got a string with `-` in it, which indicates it is a slug and handle it accordingly. If it does not have `-` then we need to create the slug and handle it accordingly. 

We want to be able to call the function two different ways `tag("hide-in-gallery")` or `tag("Hide in Gallery")`

Reply:
I tested this by sending `new-tag` and `New Tag` and both seem to work!

Reply:
ah yes yes. I guess it is still odd to me to do the explode()/implode() logic when you don't need to.

Reply:
totally ! And doesn't hurt to check - updated this!